### DISCUSSION TITLE
InnoDB 버퍼 풀 기능이란?

### DISCUSSION THREAD

**Discussion by CODe5753** [Well-Founded-Confidence/RealMySQL#6] [Q&A]
DB 성능 향상 기능 중 하나인데요,
메모리에 데이터를 캐싱하는 기능이라고 합니다.

MyISAM 키 캐시(#5)와 비교되는 기능이라고 생각하는데, 
꼭 알고 넘어가야 할 것 같아요.

**Comment by menthamin** [comment]
버퍼에 디스크에서 읽어온 비싼? 데이터를 저장해두고 캐시처럼 가져다 사용하는 것이라고 알고 있습니다.

여기에 추가로 캐시 역할을 하는데 왜 보통 DBMS에서는 버퍼풀이라고 하는지 궁금합니다.

**Comment by CODe5753** [comment]
해당 내용은 4.2.7에서 자세히 다뤄지므로 Close 합니다.
